exasol_script_languages_container_ci_setup/lib/aws/wrapper/datamodels/common.py

```diff
@@ -0,0 +1,12 @@
+import dataclasses
+from typing import Any
+
+
+@dataclasses.dataclass(frozen=True)
+class ARN:
+    aws_arn: Any
```

Comment:
What is this for?

Reply:
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html

Reply:
Sry should have formulated my question more precisely, basically I wanted to know why you are wrapping it in a `dataclass`. If it's only about typing you could consider using the [NewType](https://docs.python.org/3.8/library/typing.html#newtype)

```python
from typing import NewType

ARN = NewType('ARN', str)
```